server.elm

```diff
@@ -2156,39 +2176,140 @@ editionscript = """
+        showListsImages(srcName);
+
+        // automatically select upload image
+        let selectedImage = document.querySelectorAll(".imgFolder");
+        for (let i = 0; i < selectedImage.length; ++i) {
+          if (selectedImage[i].getAttribute("src") === files[files.length - 1]) {
+            selectedImage[i].style.outline = "2px solid white";
+          } else {
+            selectedImage[i].style.outline = "none";
+          }
+        }
       }
+
+      function showListsImages(srcName) {
+        let dir = "";
+        for(let i = 0, arr = srcName.split('\\'); i < arr.length - 1; ++i) {
+          dir += (arr[i] + "/");
+        }
+        files = defaultOptions.nodefs.listdir(dir);
+        
+        images = [];
+        files.forEach(file => {
+          let ext = file.split('.').pop();
+          if (ext == 'jpeg' || ext == 'jpg' || ext == 'png' || ext == 'gif') {
+            images.push(file);
+          }
+        });
+        
+        // init: clear image list
+        let selectedImage = document.querySelectorAll(".imgFolder");
+        selectedImage.forEach(e => e.remove());
+
+        for (let i = 0; i < images.length; ++i) {
+          keyvalues.append(
+            el("div", { class: "imgFolder", style: "width: 300px; margin: 10%" }, el("img", { style: "width: 100%", "src": dir + images[i], "alt": images[i] },  [], {}), {
+              onclick() {
+                // highlight the selected image
+                let otherImages = document.querySelectorAll(".imgFolder");
+                for (let i = 0; i < otherImages.length; ++i) {
+                  otherImages[i].style.outline = "none";
+                }
+                // replace image
+                clickedElem.setAttribute("src", this.children[0].getAttribute("src"));
+                document.getElementsByClassName("keyvalue")[0].children[1].children[0].setAttribute("value", this.children[0].getAttribute("src"));
+                this.style.outline = "2px solid white";
+              }
+            })
+          );
+        }
+      }
+
+      if (clickedElem && clickedElem.tagName === "IMG") {
+        let srcName = clickedElem.attributes[0].value;
+
+        clickedElem.ondragover = function (e) {
+          e.preventDefault();
+        }
+
+        clickedElem.ondrop = function (e) {
+          // upload and replace the image
+          e.stopPropagation();
+          e.preventDefault();
+          var files = e.dataTransfer.files; // FileList object
+          uploadImagesAtCursor(files, srcName);
+        }
+
+        // upload image button
+        keyvalues.append(
+          el("a", 
+            { style: "margin: 10%; padding: 4px 10px; height: 20px; position: relative; color: #888; background: #fafafa; border-radius: 4px; display: inline-block; height: 20px; width: 280px;" }, 
+            el(
+              "input", {"type": "file", value: "Please upload images...", style: "position: absolute; top: 0; left: 0; width: 100%;"}, 
```

Comment:
As said before, insert `id: [some unique Id` to this input so that you can address it directly.

Reply:
After you do this, put the style part somewhere in a &lt;style> tag instead of inline.